tests/test_datacard.py

```diff
@@ -33,6 +66,47 @@ def test_datacard_lines_similar(db, default_template, cache):
     assert content == default_template
 
 
+def test_datacard_player(db, cache):
+
```

Comment:
This also does not yet have a docstring.

Reply:
I added a docstring in both cases.